website/data/docs-nav-data.json

```diff
@@ -312,6 +312,7 @@
           "type": "outlined",
           "color": "neutral"
         },
+        "hidden": true,
```

Comment:
Curious what people think about this. I chose to hide the deprecated concept from the nav menu. I wouldn't normally for a feature that has been around for ages as that could cause confusion, however this feature is still in Beta, has only been in Vault for one release and is unlikely to be in use by anyone. It seemed better to just hide it to avoid the confusion and clutter of both items in the menu. Hiding it lets us keep the actual pages around with the deprecation notice in case other places deep link to them already.

This seemed like the best compromise to me but I'm open to alternative suggestions.